### PR TITLE
DF-1117 no -> cancel

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/online-services/work-efforts/work-efforts-overview/modal-config.types.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
-export const deleteWorkEffortModalConfig = {
+import { ConfirmModalConfig } from '../../../shared/layout/modal/confirm-modal/confirm-modal-config.model';
+
+export const deleteWorkEffortModalConfig: ConfirmModalConfig = {
   title: 'portal.work-efforts.work-effort.confirm-delete-modal.title',
   content: 'portal.work-efforts.work-effort.confirm-delete-modal.question',
-  confirmLabel: 'portal.work-efforts.work-effort.confirm-delete-modal.confirm.label'
+  confirmLabel: 'portal.work-efforts.work-effort.confirm-delete-modal.confirm.label',
+  cancelLabel: 'portal.work-efforts.work-effort.confirm-discard-modal'
```
Comment (99999989) [inline alv-portal-ui/src/app/online-services/work-efforts/work-efforts-overview/modal-config.types.ts]: This is not a valid translation key.

File: alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class ConfirmModalComponent {
 
   confirmLabel = 'portal.modal.confirm.yes';
 
-  cancelLabel = 'portal.modal.confirm.no';
+  cancelLabel = 'portal.modal.confirm.cancel'; // DF-1117
```
Comment (99999989) [inline alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts]: Revert this change. The issue https://alv-ch.atlassian.net/browse/DF-1117 was just about changing it for candidate-search-profiles and job-search-profiles, not generally.

File: alv-portal-ui/translations.csv [hunk 3]
```diff
@@ -742,6 +742,7 @@ portal.candidate-search-profiles.notification.profile-deleted,Ihre Suche wurde g
 portal.candidate-search-profiles.delete-confirmation-modal.title,Suche löschen?,Delete search?,Supprimer la recherche?,Cancellare la ricerca?,
 portal.candidate-search-profiles.delete-confirmation-modal.question,"Möchten Sie diese Suche ""<strong>{{profileName}}</strong>"" wirklich löschen?","Are you sure you want to delete this search ""<strong>{{profileName}}</strong>""?","Souhaitez-vous vraiment supprimer cette recherche ""<strong>{{profileName}}</strong>""?","Volete davvero cancellare questa ricerca ""<strong>{{profileName}}</strong>""?",
 portal.candidate-search-profiles.delete-confirmation-modal.button.confirm,"Ja, löschen","Yes, delete","Oui, supprimer","Sì, cancellare",
+portal.candidate-search-profiles.delete-confirmation-modal.button.cancel,Abbrechen,Cancel,Annuler,Annullare,
```
Comment (99999989) [inline alv-portal-ui/translations.csv]: This translation key is added but never used. Should be used in the `modal-config.ts` of the candidate-search-profiles.

File: alv-portal-ui/translations.csv [hunk 4]
```diff
@@ -943,7 +945,7 @@ portal.manage-job-ads.search.filter.modal.title,Stellenanzeigen filtern,Filter j
 portal.manage-job-ads.search.filter.status.all,Alle,All,Tous,Tutti,
 portal.manage-job-ads.search.owner-name,Erstellt von,Created by,Créé par,Creato da,
 portal.manage-job-ads.search.query,Stellen durchsuchen,Search jobs,Rechercher des emplois,Sfogliare i posti annunciati,
-portal.modal.confirm.no,Nein,No,Non,No,
```
Comment (99999989) [inline alv-portal-ui/translations.csv]: Revert this change

File: alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts (hunk 2 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts]: I clarified it with Ingo. The default label for confirmation dialogs should me "Cancel", "Yes" instead of "No", "Yes"

File: alv-portal-ui/translations.csv (hunk 4 above)
Comment (ganqqwerty) [reply alv-portal-ui/translations.csv]: see above

File: alv-portal-ui/src/app/online-services/work-efforts/work-efforts-overview/modal-config.types.ts (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/online-services/work-efforts/work-efforts-overview/modal-config.types.ts]: ouch, yes, my bad. fixed

Comment (99999989) [reply alv-portal-ui/src/app/online-services/work-efforts/work-efforts-overview/modal-config.types.ts]: It's still the wrong translation key (or just remove it to leverage the default value)

File: alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/shared/layout/modal/confirm-modal/confirm-modal.component.ts]: ok

File: alv-portal-ui/translations.csv (hunk 4 above)
Comment (99999989) [reply alv-portal-ui/translations.csv]: ok
